tests/src/test/scala/org/apache/openwhisk/core/limits/ActionLimitsTests.scala

```diff
@@ -66,7 +66,7 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers with WskActorSys
 
   val openFileAction = TestUtils.getTestActionFilename("openFiles.js")
   val openFileLimit = 1024
-  val minExpectedOpenFiles = openFileLimit - 15 // allow for already opened files in container
+  val minExpectedOpenFiles = openFileLimit - 20 // allow for already opened files in container
```

Comment:
is this necessary?

Reply:
The nodejs:10 runtime has some more (+5) already open file handles which reduces the remaining number of open files by 5. The test is `successfully invoke an action when it is within nofile limit` therefore it seems not really necessary to always test for the exact maximum of files that can be opened. With the `-20` the test now runs fine with both versions of the nodejs runtimes.